### PR TITLE
fix(build): 💊 make Android build script cross platform

### LINKED ISSUE #1670: Error when building Android on Windows
**Describe the issue**
build with error

**Environment**
 - OS: windows10
 - NPM version (`9.8)
 - Node.js version(18.16)

**Additional context**
I am building an Android application on Windows, but an error occurred：
  
▶ action(cordova/build):
  ▶ action(www/build):
  ▶ action(www/build): � Success! (27308ms)
  ▶ action(cordova/setup):
    ▶ action(www/build):
    ▶ action(www/build): � Success! (27184ms)
    Copying Android third party libraries...
  ERROR(spawn_stream): node --trace-uncaught E:\vpnnnnnn\outline-client-client_android\src\cordova\build.action.mjs android failed w
ith exit code 1. Printing stderr:
    ERROR(spawn_stream): node --trace-uncaught E:\vpnnnnnn\outline-client-client_android\src\cordova\setup.action.mjs android failed
 with exit code 1. Printing stderr:
    �����﷨����ȷ��
  ▶ action(cordova/setup): ❌ Failed.
▶ action(cordova/build): ❌ Failed.

### PR DESCRIPTION
**Pull request by jyyi1**
This PR fixes #1670 .

### CODE REVIEW

File: src/cordova/plugin/android/scripts/copy_third_party.js [hunk 1]
```diff
@@ -14,15 +14,16 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-const child_process = require('child_process');
+const fs = require('fs');
```
[the remark this answers is not in the document]
Comment (jyyi1) [reply src/cordova/plugin/android/scripts/copy_third_party.js:17]: hmm, not a bad idea.

File: src/cordova/plugin/android/scripts/copy_third_party.js [hunk 2]
```diff
@@ -14,15 +14,16 @@
...
   console.log('Copying Android third party libraries...');
-  child_process.execSync('mkdir -p plugins/cordova-plugin-outline/android/libs');
-  child_process.execSync(
-    `cp third_party/outline-go-tun2socks/android/tun2socks.aar plugins/cordova-plugin-outline/android/libs/`
-  );
-  child_process.execSync(
-    `cp -R third_party/outline-go-tun2socks/android/jni plugins/cordova-plugin-outline/android/libs/obj`
+  fs.mkdirSync('plugins/cordova-plugin-outline/android/libs', {recursive: true});
+  fs.copyFileSync(
+    'third_party/outline-go-tun2socks/android/tun2socks.aar',
```
[the remark this answers is not in the document]
Comment (jyyi1) [reply src/cordova/plugin/android/scripts/copy_third_party.js:23]: You mean `join` the paths? Yes sure.

**Comment by jyyi1** [comment]
@daniellacosse BTW, what do I need to do to deal with these code coverage errors?

File: src/cordova/plugin/android/scripts/copy_third_party.js [hunk 3]
```diff
@@ -14,15 +14,20 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-const child_process = require('child_process');
+const fs = require('fs/promises');
+const path = require('node:path');
 
-module.exports = function(context) {
+const ANDROID_LIBS_FOLDER_PATH = 'plugins/cordova-plugin-outline/android/libs';
```
[the remark this answers is not in the document]
Comment (jyyi1) [reply src/cordova/plugin/android/scripts/copy_third_party.js]: ah yes, this can make the path more cross-platform. updated.

**Review by fortuna — APPROVED** [review]

**Pull request merged**